### DISCUSSION TITLE
Exclude tags in templates folder from showing in tag pane?

### DISCUSSION THREAD

**Discussion by Keysz** [BryanJenksCommunity/FAQ#72] [Q&A] [answered]
Hi Bryan, 

I'd like my tag pane and search overall to exclude tags mentioned in my template folder. That way I can just focus on relevant notes to work on. Is there a way I can filter these out? Also, same question for excluding these from the graph view? 

Thanks!

**Comment by tallguyjenks** [comment, marked as answer]
excluding in tag pane off the top of my head I don't know a way.

for the graph you can use the graph search `-path:Templates` or another thing I do is sometimes change the color of nodes via the color groups to be the same color as the graph background so they disappear except the edges that connect nodes together
